tst/teststandard/bugfix.tst

```diff
@@ -1082,8 +1082,8 @@ gap> StructureDescription(testG(8,2));
 "((C8 x C2) : C2) : C2"
 gap> StructureDescription(testG(8,3));
 "C3 x QD16"
-gap> StructureDescription(testG(8,4));
-"((C16 x C2) : C2) : C2"
+gap> Length(StructureDescription(testG(8,4)));
```

Comment:
What about 
```
gap> StructureDescription(testG(8,4)) in [ "((C16 x C2) : C2) : C2", "(C16 x C4) : C2" ];
true
```
Not that I like this too much, but it does not contradict to a documented behaviour.